site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+
+<div class="offcanvas-start offcanvas-lg d-flex" role="region" aria-labelledby="sideNavCwdc1Toggle" id="sideNavCwdc1">
+  <div class="offcanvas-header p-0 side-nav-content-with-drawer">
+    <button type="button" class="btn-close ms-auto" data-bs-dismiss="offcanvas" data-bs-target="#sideNavCwdc1" aria-expanded="true" aria-controls="sideNavCwdc1"><span class="visually-hidden">Close</span></button>
+  </div>
+  <div class="offcanvas-body p-0">
+
+    <nav class="side-nav" aria-label="Collapsible side navigation with drawer content inside an offcanvas">
+      <div class="side-nav-content-with-drawer">
+        <button class="side-nav-toggle collapsed" data-bs-toggle="collapse" data-bs-target="#sideNavCollapseCwdc1"><span class="visually-hidden">Open/Close side navigation</span></button>
+        <div class="side-nav-content">
+          <ul class="nav d-block h-100 list-unstyled m-0 side-nav-dark side-nav-scrollable" role="menubar">
+            <li class="nav-item" role="none" data-bs-toggle="tooltip" data-bs-title="Dashboard" data-bs-placement="right">
+              <button class="side-nav-item active" data-bs-toggle="tab" data-bs-target="#dashboardTab1" type="button" role="menuitem" aria-controls="dashboardTab1" aria-expanded="false" aria-haspopup="true" onclick="boosted.Collapse.getOrCreateInstance('#sideNavCollapseCwdc1').show()">
+                <svg class="align-self-center" width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#dashboard"/>
+                </svg>
+                <span class="visually-hidden">Open dashboard tab</span>
+              </button>
+            </li>
+            <li class="nav-item" role="none" data-bs-toggle="tooltip" data-bs-title="Charts" data-bs-placement="right">
+              <button class="side-nav-item" data-bs-toggle="tab" data-bs-target="#chartsTab1" type="button" role="menuitem" aria-controls="chartsTab1" aria-expanded="false" aria-haspopup="true" onclick="boosted.Collapse.getOrCreateInstance('#sideNavCollapseCwdc1').show()">
+                <svg class="align-self-center" width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#chart"/>
+                </svg>
+                <span class="visually-hidden">Open charts tab</span>
+              </button>
+            </li>
+            <li class="nav-item" role="none" data-bs-toggle="tooltip" data-bs-title="Financial" data-bs-placement="right">
+              <button class="side-nav-item" data-bs-toggle="tab" data-bs-target="#financialTab1" type="button" role="menuitem" aria-controls="financialTab1" aria-expanded="false" aria-haspopup="true" onclick="boosted.Collapse.getOrCreateInstance('#sideNavCollapseCwdc1').show()">
```

Comment:
Remove inline JS.

Reply:
Not sure about my comment, it may be the easiest way to explain the code example? I need to think about it... 🤔

Reply:
I need to add those until we have some proper Js otherwise, we wouldn't see the expected behavior of this component. As said in design review, depending on the a11y returns, the Js might be done in a standalone PR after to ease the review process.